packages/components-providers/src/I18n/resources.ts

```diff
@@ -0,0 +1,120 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { Resource } from 'i18next'
+
+export const i18nResources: Resource = {
+  en: {
+    AdvancedInputControls: {
+      Label: 'Clear Field',
+    },
+    Avatar: {
+      Name: 'Avatar',
+    },
+    BulkActions: {
+      Button: 'Bulk Actions',
+      SelectedAll: 'Select all {{totalCount}} results',
+      SelectedItem: 'Clear Selection',
+      SelectedItemCountEqualToPageCount:
+        'All {{pageCount}} displayed items selected',
+      SelectedItemCountEqualToTotalCount: 'All {{totalCount}} items selected',
+      SelectedItemCountLessThenPageCount:
+        '{{selectedItemCount}} of {{pageCount}} displayed items selected',
+    },
+    CalendarNav: {
+      Content: 'View Current Month',
+    },
+    CheckMarkMixed: {
+      CheckMark: 'Check Mark Mixed',
+    },
+    Chip: {
+      Delete: 'Delete',
+    },
+    ColumnSelector: {
+      Apply: 'Apply',
+      Cancel: 'Cancel',
+      Label: 'Select columns to display',
+      SelectAll: 'Select All',
+      SelectNone: 'Select None',
+    },
+    ConfirmationDialog: {
+      CancelLabel: 'Cancel',
+      ConfirmLabel: 'Confirm',
+    },
+    CopyToClipboard: {
+      Children: 'Copy to Clipboard',
+      Success: 'Copied',
+    },
+    DataTableItem: {
+      ActionsTooltip: 'Options',
+    },
+    GetIntentLabel: {
+      Error: 'Error',
+      Inform: 'inform',
+      Positive: 'positive',
+      Warn: 'Warning',
+    },
+    InputDate: {
+      Placeholder:
+        'Date {{formatDateString(new Date(Date.now())}}, {{dateStringLocale}}',
```

Comment:
I think this one can break the rule we chatted about earlier of not breaking up phrases because of languages like Japanese – since it's not really a phrase, more like a label (see suggestion part 2 in `InputDate.tsx`):
```suggestion
      Date: 'Date',
```
![image](https://user-images.githubusercontent.com/53451193/107576436-fb9bce00-6ba5-11eb-8b21-52f17cd22ed6.png)